### DISCUSSION TITLE
Getting @load to work on q-carousel slide

### DISCUSSION THREAD

**Discussion by 406rend** [quasarframework/quasar#13032] [General - Components / Directives / etc] [answered]
I am trying to get @load to fire on a q-carousel slide. The goal being to show a spinner or change the visibility of the carousel until the images load. Nothing I have done has been able to get the function to fire @load. 

The carousel is one of many within a page, which will be activated only when its parent div is clicked, so I would not want to force all images to load when mounted.

      <q-carousel
          swipable
          animated
          v-model="slider[i]"   
        >
          <q-carousel-slide
            v-for="img in imgPaths"
            :key="img.no"
            :name="img.no"
             id="slider"
            @load="imgIsLoaded('slider')"
            :img-src="img.url"
          />
        </q-carousel>

**Comment by MilosPaunovic** [comment, marked as answer]
From the docs page, `QCarouselSlide` does not emit `@load` event. You can use `default` slot and do your custom logit in there.

**Reply by 406rend** [reply]
Thanks very much. I don't yet know how to work with logic in a default slot, but I'll learn!
